### ISSUE TITLE
Add notification periods

### ISSUE THREAD

**Issue by shanemd** [AllanWang/Frost-for-Facebook#1711]
The minimum setting for Frost to check for notifications is 15 minutes. It would be nice to have 5 minute and 10 minute options as well. This would be helpful for messages in particular. I imagine there could be issues with this that I'm not aware of though. (API rate limits maybe?)

Also, thank you for building such a useful app! This is a massive improvement in user experience over the prescriptive and manipulative official Facebook app (Why can't I see the full notification text in the Android notification?!). Not to mention how I just freed up ~500MB of storage space, and who knows how many CPU cycles. Many thanks!

**Comment by AllanWang** [comment]
See #1358, https://www.reddit.com/r/FrostForFacebook/comments/h7c9kn/real_time_notifications/

This is answered in the app faq, but I will need to link it elsewhere since it's overlooked

**Comment by shanemd** [comment]
Ah, thanks @AllanWang. Thanks for the pointer to the FAQ. It took me a while to find it. It's not linked in the README (The first place I looked), or as a first-class item in the app menu. The biggest hurdle though, once I saw the word 'FAQ' in the subtext for 'About Frost for Facebook', was realizing that the about pane is paged. I accidentally discovered it by randomly swiped right. Perhaps using a tabbed selector (not sure this is the correct name for this) for the about pane would help? One of those that shows 'About     Credits     FAQ' on the top, and highlights the one you're currently viewing.

Thanks for the answer. I appreciate the battery life, and the decision to maintain a simpler codebase.

Cheers
Shane
